src/routers/datatypeRouters/nodes/proteins.ts

```diff
@@ -10,13 +10,15 @@ const schema = loadSchemaConfig()
 
 export const proteinsQueryFormat = z.object({
   name: z.string().optional(),
+  full_name: z.string().optional(),
   dbxrefs: z.string().optional(),
   page: z.number().default(0)
 })
 
 export const proteinFormat = z.object({
   _id: z.string(),
   name: z.string(),
+  full_name: z.string(),
```

Comment:
do all proteins have a 'full_name' defined? just checking if this field should be optional or not

Reply:
emm, I will change it to optional